backend/src/modules/doubles-tournament/core/application/event/EndTournamentMatchWhenMatchHasEnded.ts

```diff
@@ -0,0 +1,21 @@
+import { EventHandler } from '../../../../../shared/core/application/event/EventHandler';
+import { MatchHasEnded } from '../../../../match-module/core/domain/event/MatchHasEnded';
+import { DomainEventPublisher } from '../../../../../shared/core/application/event/DomainEventBus';
+import { CurrentTimeProvider } from '../../../../../shared/core/CurrentTimeProvider';
+import { TournamentMatchWasEnded } from '../../domain/event/TournamentMatchWasEnded';
+
+export class EndTournamentMatchWhenMatchHasEnded implements EventHandler<MatchHasEnded> {
```

Comment:
Moze nazwa: 
`MatchHasEndedImpliesTournamentMatchWasEnded`
Do przemyslania. Bo teraz mi to sugeruje wywolanie commanda. Ale nie koniecznie :) 
Zastanawia mnie w sumie czemu w Match mamy Has, jak wszedzie Was. Ale to moze nie na teraz romzowy :)

Reply:
A to nie wiem, inny moduł - inny język ;)